game/static/game/js/game.js

```diff
@@ -588,9 +598,40 @@ ocargo.Game.prototype._setupPythonTab = function () {
         $('#consoleOutput').text('');
     }.bind(this));
 
+    var leadMsg = '<p>' + interpolate(gettext('Run the following commands on the van object %(var_name)s, e.g. %(example)s'),
```

Comment:
I'm not sure what should be included in translation string. In this case I wrapped as less as possible to avoid mistakes.